src/foam/nanos/auth/User.js

```diff
@@ -16,7 +16,7 @@ foam.CLASS({
   documentation: '',
 
   tableColumns: [
-    'id', 'enabled', 'firstName', 'lastName', 'organization', 'lastModified'
+    'id', 'enabled', 'firstName', 'lastName', 'organization', 'lastModified', 'profilePicture'
```

Comment:
I'm not too keen on having the provilePicture be a column in the table view, but people can hide that column if they want, so I guess it's okay.

Reply:
It was just for testing purposes. It can be removed.